### ISSUE TITLE
Feature request: Adjust brightness and contrast of images

### ISSUE THREAD

**Issue by moi90** [ecotaxa/ecotaxa_front#861]
Sometimes, images are easier to annotate if their contrast is stretched (for example). However, this currently requires altering the image file.

I propose a small UI to change brightness and contrast of the displayed images. This can be purely done with JS and CSS:

https://www.w3schools.com/cssref/css3_pr_filter.php

If you point me in the right direction, I might start working on it.

**Comment by moi90** [comment]
I created a fiddle to show the functionality:

https://jsfiddle.net/6qmpt40n/23/

![grafik](https://github.com/ecotaxa/ecotaxa_front/assets/332746/902a4ba2-7b1e-4236-a072-4a8a0067dfdd)

As EcoTaxa already uses JQuery, it should be easy to include this.

**Comment by grololo06** [comment]
As all images in a page are likely to have the same need for improvement (and we have big CPUs nowadays), I think it would make sense to have this _globally_ for all visible objects in a classification page, and with Brightness/Contrast controls always visible. Users won't bother clicking a button then adjusting then closing.

**Comment by moi90** [comment]
> I think it would make sense to have this globally for all visible objects in a classification page

Yes, exactly what I was thinking! (Not for each individual object, that would be stupid.)

> with Brightness/Contrast controls always visible. Users won't bother clicking a button then adjusting then closing.

Is there enough space on the screen? It could also be a menu (that opens on hover) with the sliders. Or two simple dropdowns (analogous to, and next to, the magnification dropdown). The nice thing about the sliders is that the effect on the images is instant, while dragging.

**Comment by bea777515** [comment]
Closed -> moved on clickup - will be a feature of the new classification page (similar but tailwindcss - vanilla js) , interactions to be clarified

**Comment by moi90** [comment]
Great that you will be working on it! Is the Clickup publicly viewable? (Otherwise, it might make sense to keep this issue open until the feature is actually implemented.)

**Comment by bea777515** [comment]
re-opened ,  ideas about the controls and interactions can be clarified here

**Comment by bea777515** [comment]
I am closing all the issues , and moving them to clickup , except those to be clarified , and, you are right, this one should stay here to add ideas about the UI

**Comment by juliecoust** [comment]
Déplacé sur clickup
